frontend/ui/lib/widgets/place_block_widget.dart

```diff
@@ -24,8 +24,13 @@ class _PlaceBlockWidgetState extends State<PlaceBlockWidget> {
             child: Row(children: [
               ToggleButtons(
                 children: [
-                  Tooltip(message: "This selection makes the place a MUST GO!", child: Text('M')),
-                  Tooltip(message: "This selection makes the place optional for if TIME PERMITS.", child: Text('T')),
+                  Tooltip(
+                      message: "This selection makes the place a MUST GO!",
```

Comment:
Nit: consider making these tooltips shorter, e.g., "Must go" and "If time permits"